cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -19,6 +19,7 @@ package kubeadm
 import (
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	bootstraptokenutil "k8s.io/cluster-bootstrap/util/tokens"
```

Comment:
the k8s.io/cluster-bootstrap/util/tokens package is not intended to define config APIs for decomposed bootstrap tokens